src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -114,6 +128,21 @@ public void Dispose()
             _http.Dispose();
         }
 
+        private string GetXrefMapApiEndpoint(string xrefEndpoint)
+        {
+            var isProduction = string.Equals(xrefEndpoint?.TrimEnd('/'), "https://xref.docs.microsoft.com", StringComparison.OrdinalIgnoreCase);
```

Comment:
If `xrefEndPoint` is not set, should it respect the current `DOCS_ENVIRONMENT` setting?

Reply:
yes